versions/3.0.4.md

```diff
@@ -1080,8 +1080,8 @@ form | false | color= | color=blue | color=blue,black,brown | color=R,100,G,200,
 form | true | color= | color=blue | color=blue&color=black&color=brown | R=100&G=200&B=150
```

Comment:
Should this have the name too?
```suggestion
form | true | color= | color=blue | color=blue&color=black&color=brown | color=R=100&G=200&B=150
```

Reply:
Also, the lines above, which I cannot comment on, seems to be missing parameter name in the "object" column.

[`style`](#dataTypeFormat) | `explode` | `empty` | `string` | `array` | `object`
----------- | ------ | -------- | -------- | -------- | -------
matrix | true | ;color | ;color=blue | ;color=blue;color=black;color=brown | ;color=R=100;G=200;B=150

Reply:
I think objects are "exploded" by using their members as query parameter names, see [Parameter Serialization Control](https://learn.openapis.org/specification/parameters.html#parameter-serialization-control), so the current text is what I would expect.